docs/develop/java/versioning.mdx

```diff
@@ -188,27 +220,3 @@ You can also promote an entire set to become the default set for the queue. New
 workflowClient.updateWorkerBuildIdCompatability(
```

Comment:
```suggestion
workflowClient.updateWorkerBuildIdCompatibility(
```

Hopefully that is just a typo in the code sample and not the API itself.

Reply:
Alas, it's in the API.  But luckily, this is the old API that's going away.